examples/compose/plot_compare_reduction.py

```diff
@@ -19,13 +19,14 @@
 
 Note that the use of ``memory`` to enable caching becomes interesting when the
 fitting of a transformer is costly.
+"""
 
 # %%
-Illustration of ``Pipeline`` and ``GridSearchCV``
+# Illustration of ``Pipeline`` and ``GridSearchCV``
 ###############################################################################
 
-This section illustrates the use of a ``Pipeline`` with ``GridSearchCV``
-"""
+# This section illustrates the use of a ``Pipeline`` with ``GridSearchCV``
+
```

Comment:
This fix is not related to the original PR. I think the recommendation at https://github.com/scikit-learn/scikit-learn/pull/18758/files#r596937724 was to open another PR with only this fix.